target/scripts/helpers/index.sh

```diff
@@ -1,16 +1,25 @@
 #! /bin/bash
+
 # shellcheck source-path=target/scripts/helpers
 # This file serves as a single import for all helpers
 
+export CHKSUM_FILE=/tmp/docker-mailserver-config-chksum
```

Comment:
Why was `export` added?

Reply:
While not strictly needed, other scripts like `start-mailserver.sh` depend on this variable. The `export` statement shows this explicitly.

Reply:
> While not strictly needed, other scripts like start-mailserver.sh depend on this variable

If that's the main concern, then a comment would be better. + In the comment you could name the script that depends on it.

It triggers me each time I see a _useless_ export 😆

Reply:
Alright, if it is indeed useless, I agree, and I will drop it and add a comment instead :) I looked at the help message of `export` and you're right, it is not needed here :D

Reply:
ref f8e77a26257c26823a4b57f0375547c99ec75dc0

Reply:
The same goes for the `export -f` statements?

General / for clarification: All of our scripts sources helper files, if they need a specific function or variable. So there should never be a need to export anything.
Non of our scripts does expect a function or variable just to be there, without sourcing a helper script.

Reply:
I don't think the whole line does serve any purpose: https://github.com/docker-mailserver/docker-mailserver/blob/ec8b99335ea3d1d9754217cf4826582caf0c0bec/target/scripts/start-mailserver.sh#L74

Reply:
I removed all the `export -f` statements from the helpers and removed the `export` line from `start-mailserver.sh`.

Reply:
One last thing: You removed the `export -f` but not the whole line. Now there is an extra empty line.

Reply:
PS:

> I don't think the whole line does serve any purpose: 

I hope you can confirm this and not just removed it because I said so.

Reply:
> One last thing: You removed the `export -f` but not the whole line. Now there is an extra empty line.

I removed them.

---

> PS:
> 
> > I don't think the whole line does serve any purpose:
> 
> I hope you can confirm this and not just removed it because I said so.

I have double-checked what you said and was able to confirm that you're correct. I then went ahead and removed the statements.